geospaas_harvesting/ingesters.py

```diff
@@ -310,35 +319,53 @@ def _get_xml_namespace(self, root):
             self.LOGGER.warning('Could not find XML namespace while reading DDX metadata')
         return namespace_prefix
 
-    def _extract_global_attributes(self, root):
-        """Extracts the global attributes of a dataset from a DDX document"""
+    def _extract_attributes(self, root):
+        """Extracts the global or specific attributes of a dataset or specific ones from a DDX document
+        global_flag=True means searching for global attributes
+        OTHERWISE
+        global_flag=False means searching for specific attributes"""
```

Comment:
This is not clear. What is global_flag?

Reply:
Sorry, that docstring belongs to the previous version of implementation of this method. I forgot to update the docstring after completely rewrite this function. You will find the correct docstring in the next commit.

Done in the next commit.